draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -766,11 +766,11 @@ The format of this uplink is application specific.  It is RECOMMENDED for a
 device to send an empty frame (see {{lorawan-empty-frame}}) but it is application
 specific and will be used by the NGW to transmit a potential SCHC ACK REQ.  
 SCHC_ACK_REQ_DN_OPPORTUNITY is application specific and its recommended value
-is 2, it MUST be greater than 1. This allows to open downlink opportunity to other
-eventual downlink with higher priority than SCHC ACK REQ message.
+is 2. It MUST be greater than 1. This allows to open a downlink opportunity to
+any downlink with higher priority than the SCHC ACK REQ message.
 
 _Note_: The device MUST keep this SCHC ACK message in memory until it receives
-a downlink, on SCHC FPortDown different from an SCHC ACK REQ: it indicates that
+a downlink with a SCHC FPortDown different from that of the SCHC ACK REQ: it indicates that
```

Comment:
Are you sure regarding `that of the` ?

Reply:
until it receives a downlink SCHC Fragmentation Message (with FPort == FPortDown) that is not a SCHC ACK REQ.